identity/generate.go

```diff
@@ -45,14 +50,28 @@ func GetPublicIdentity(b64Identity string) (*string, error) {
 		return nil, err
 	}
 
-	if publicIdentity.Target != "user" && publicIdentity.Target != "email" {
+	if publicIdentity.Target != "user" &&
+		publicIdentity.Target != "email" &&
+		publicIdentity.Target != "phone_number" {
```

Comment:
Will it not break with `hashed_email`?

Reply:
hashed_email is only in the private identity, so here it's always `email`